### PR TITLE
Add setting to hide taglines on movie detail pages

### PR DESCRIPTION
**Pull request by 1hitsong**
**Changes**
Creates new user setting that allows for toggling visible/hidden of the movie tagline text content. Setting comes from a request posted on reddit.

https://www.reddit.com/r/jellyfin/comments/vezhz6/disable_taglines/

### CODE REVIEW

**Review by candry7731 — APPROVED** [review]
Works as Intended
No issues found

File: components/tvshows/TVShowDetails.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <LayoutGroup id="toplevel" layoutDirection="vert" itemSpacings="[-10]" >
       <LayoutGroup id="main_group" layoutDirection="horiz" itemSpacings="[15]" >
         <Poster id="tvshowPoster" width="300" height="450" />
-        <LayoutGroup layoutDirection="vert" itemSpacings="[15]">
+        <LayoutGroup layoutDirection="vert" itemSpacings="[15]" id="details">
```
Comment (jimdogx) [inline components/tvshows/TVShowDetails.xml]: Only question, I noticed you added the ID for TV details as well, but I only see code that hides taglines for Movies.

**Review by jimdogx — CHANGES_REQUESTED** [review]

File: components/tvshows/TVShowDetails.xml (hunk 1 above)
Comment (1hitsong) [reply components/tvshows/TVShowDetails.xml]: I originally had code to hide taglines on TV shows, but it turned out we didn't populate the value. Can you cherrypick around this, or do you need me to update the PR?

Comment (jimdogx) [inline components/tvshows/TVShowDetails.xml]:
```suggestion
        <LayoutGroup layoutDirection="vert" itemSpacings="[15]">
```
Even easier, we can just fix it "inline" right here in the PR 😄

**Review by jimdogx — APPROVED** [review]

**Pull request merged**